### PR TITLE
[PS-2311] Added notes about Visual Studio 2022 requirement for Windows developers.

### PR DESCRIPTION
**Pull request by thlucas1**
BW Server project would not compile using Visual Studio 2019 due to NET 6.0 SDK restrictions. Suggested by Thomas Rittsen @eliykat via gitter.im BW Lobby.

## Objective

Contributor documentation edits for BitWarden Server Setup guide.

### CODE REVIEW

File: docs/getting-started/server/guide.mdx [hunk 1]
```diff
@@ -18,11 +18,13 @@ Before you start: make sure you’ve installed the recommended
...
 
 - Docker Desktop
-- Visual Studio
+- Visual Studio **
 - Powershell
 - [NET 6.0 SDK](https://dotnet.microsoft.com/download)
 - Azure Data Studio
 
+** If developing in a Windows environment, Visual Studio 2022 is required in order to compile with NET 6.0 SDK framework.  See [this article](https://learn.microsoft.com/en-us/dotnet/core/compatibility/sdk/6.0/vs-msbuild-version) for more details.
+
```
Comment (eliykat) [inline docs/getting-started/server/guide.mdx:27]: This isn't a Windows-specific requirement - Visual Studio 2022 is also available for Mac, so I assume it's required for .NET 6 in that environment as well. Let's keep it simple, please delete this note and just update the dot-point to specify `Visual Studio 2022`.

**Review by eliykat — CHANGES_REQUESTED** [review]
Thanks for following up on this @thlucas1!

**Comment by thlucas1** [comment]
Updated the server guide to suggested changes.

File: docs/getting-started/server/guide.mdx (hunk 1 above)
Comment (thlucas1) [reply docs/getting-started/server/guide.mdx]: Thanks Thomas.  I was a Windows developer by trade (retired now), so should have realized the requirements for Mac would be the same.  I updated the server guide to your suggested changes.

**Review by eliykat — APPROVED** [review]
Thank you!

**Pull request merged**